Task_class_1/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
the files inside the ".idea" folder shouldn't be upload as they are configurations files for Pycharm, you can add a `.gitignore` file to avoid upload it again

Reply:
Added `.idea` folder to `.gitignore` file.